hamilton/function_modifiers/macros.py

```diff
@@ -870,3 +872,106 @@ def optional_config(self) -> Dict[str, Any]:
 #
 #     def __init__(self, *transforms: Applicable, collapse=False):
 #         super(flow, self).__init__(*transforms, collapse=collapse, _chain=False)
+
+
+class loader(base.NodeCreator):
```

Comment:
This calls with `@loader()`. 

Do you ever foresee having parameters? Or should be make it so `loader=Loader()` and have it all be a static instance? Then we call it with `@loader`.

Reply:
`target_` ?